### PR TITLE
feat(locale): implement state info on sk locale

### LINKED ISSUE #3052: faker.location.state() not working for sk
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

I'm trying to get `faker.local.state()` for local sk with a local fallback en_US, but I keep getting this error `[Error]: The locale data for 'location.state' aren't applicable to this locale.`

### Minimal reproduction code

```js
const {sk, Faker, en_US} = require('@faker-js/faker')

const faker = new Faker({locale: [sk, en_US]})
console.log(faker.location.state())
```

### Additional Context

image

### Environment Info

```shell
System:
    OS: macOS 14.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 291.94 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.16.0 - ~/.nvm/versions/node/v20.16.0/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v20.16.0/bin/yarn
    npm: 10.8.1 - ~/.nvm/versions/node/v20.16.0/bin/npm
  Browsers:
    Chrome: 127.0.6533.100
    Safari: 17.5
  npmPackages:
    @faker-js/faker: 8.4.1 => 8.4.1
```

### Which module system do you use?

- [x] CJS
- [ ] ESM

### Used Package Manager

npm

### PR DESCRIPTION
**Pull request by boubkerbribri**
### Issue Number
#3052 

### Changes Made
- Adding Missing SK states following [Regions_of_Slovakia](https://sk.wikipedia.org/wiki/Kraj_(štátna_správa))

### CODE REVIEW

**Comment by xDivisionByZerox** [comment]
@boubkerbribri you need to adjust the test in `test/all-functional.spec.ts`. You need to remove sk from the "broken locale" dictionary: 

https://github.com/faker-js/faker/blob/bb1a73ca3a0ac3b3cf7a0ec25740bf6e872887ae/test/all-functional.spec.ts#L53

Additionally, you should run the [`preflight`](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md#important) command before committing to verify your changes locally.

**Comment by boubkerbribri** [comment]
> @boubkerbribri you need to adjust the test in `test/all-functional.spec.ts`. You need to remove sk from the "broken locale" dictionary:
> 
> https://github.com/faker-js/faker/blob/bb1a73ca3a0ac3b3cf7a0ec25740bf6e872887ae/test/all-functional.spec.ts#L53
> 
> Additionally, you should run the [`preflight`](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md#important) command before committing to verify your changes locally.

Should be all good now, thanks

File: src/locales/sk/location/state.ts [hunk 1]
```diff
@@ -1 +1,10 @@
-export default null;
+export default [
+  'Bratislava',
+  'Trnava',
+  'Trenčín',
+  'Nitra',
+  'Žilina',
+  'Banská Bystrica',
+  'Prešov',
+  'Košice',
```
Comment (matthewmayer) [inline src/locales/sk/location/state.ts]: Shouldnt these use the Slovak spellings? https://sk.wikipedia.org/wiki/Kraj_(%C5%A1t%C3%A1tna_spr%C3%A1va)

Comment (boubkerbribri) [reply src/locales/sk/location/state.ts]: you're right, it should

Comment (boubkerbribri) [reply src/locales/sk/location/state.ts]: done thanks

File: src/locales/sk/location/state.ts [hunk 2]
```diff
@@ -1 +1,10 @@
-export default null;
+export default [
+  'Bratislavský',
+  'Trnavský',
+  'Trenčiansky',
+  'Nitriansky',
+  'Žilinský',
+  'Banskobystrický',
+  'Prešovský',
+  'Košický',
```
Comment (matthewmayer) [inline src/locales/sk/location/state.ts]: Please also sort alphabetically (in some modules we enforce this. We don't on location module yet but we likely will in future, so this saves unneccessary resorting later)

Comment (boubkerbribri) [reply src/locales/sk/location/state.ts]: done

**Review by ST-DDT — APPROVED** [review]

**Review by xDivisionByZerox — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]
❤️

**Pull request merged**